DynmapCore/src/main/java/org/dynmap/DynmapCore.java

```diff
@@ -140,7 +140,7 @@
     public boolean is_reload = false;
     public static boolean ignore_chunk_loads = false; /* Flag keep us from processing our own chunk loads */
 
-    private MarkerAPIImpl   markerapi;
+    public MarkerAPIImpl   markerapi;
```

Comment:
I'm not sure about making this a public field. It could cause other plugin devs to access it instead of the main API.

Reply:
you are probably right on that one. I will have a look into it and see if I can come up with a different solution, thats not changing too much of the core code.